examples/filtering-ticket-groups/main.js

```diff
@@ -0,0 +1,172 @@
+  minPrice.setAttribute('min', minPriceFilter)
+  minPrice.setAttribute('max', maxPriceFilter)
+  minPrice.setAttribute('value', minPriceFilter)
+
+  const maxPrice = document.querySelector('[name="max-price"]')
+  maxPrice.setAttribute('min', minPriceFilter)
+  maxPrice.setAttribute('max', maxPriceFilter)
+  maxPrice.setAttribute('value', maxPriceFilter)
+}
+
+const updatePriceDisplays = () => {
+  const minPrice = document.getElementById('min-price')
+  minPrice.innerHTML = ''
+  minPrice.appendChild(document.createTextNode(`$${minPriceFilter}`))
+
+  const maxPrice = document.getElementById('max-price')
+  maxPrice.innerHTML = ''
+  maxPrice.appendChild(document.createTextNode(`$${maxPriceFilter}`))
+}
+
+/**
+ * Filtering
+ */
+
+const filterBySection = (selectedSections) =>
+  availableTicketGroups.filter(ticketGroup =>
+    selectedSections.length === 0 || selectedSections.includes(ticketGroup.tevo_section_name))
+
+const filter = () =>
+  allTicketGroups
+    .filter(ticketGroup => ticketGroup.quantity >= quantityFilter)
+    .filter(ticketGroup => ticketGroup.retail_price <= maxPriceFilter)
+    .filter(ticketGroup => ticketGroup.retail_price >= minPriceFilter)
+    .filter(ticketGroup => typeFilter.size === 0 || typeFilter.has(ticketGroup.format))
+
+/**
+ * Filter Listeners
+ */
+
+const update = () => {
+  availableTicketGroups = filter()
+  updatePriceDisplays()
+  map.updateTicketGroups(availableTicketGroups)
+  renderTicketGroupList(availableTicketGroups)
+}
+
+const updateFilters = (name, value) => {
+  switch (name) {
+    case 'quantity':
+      quantityFilter = value
+      break
+    case 'min-price':
+      minPriceFilter = value
+      break
+    case 'max-price':
+      maxPriceFilter = value
+      break
+    case 'type':
+      if (typeFilter.has(value)) {
+        typeFilter.delete(value)
+      } else {
+        typeFilter.add(value)
+      }
+      break
+    default:
+  }
+}
+
+document.getElementById('filters')
+  .addEventListener('change', ({ target: { name, value } }) => {
+    updateFilters(name, value)
+    update()
+  })
+
+const ticketGroupsElement = document.getElementById('ticketGroups')
+ticketGroupsElement.addEventListener('mouseover', (event) => {
+  const element = event.path.find((path) => path.classList && path.classList.contains('ticket-groups-list-item'))
+  if (element) {
+    map.highlightSection(element.dataset.section)
+  }
+})
+ticketGroupsElement.addEventListener('mouseout', (event) => {
+  const element = event.path.find((path) => path.classList && path.classList.contains('ticket-groups-list-item'))
+  if (element) {
+    map.unhighlightSection(element.dataset.section)
+  }
+})
+
+/**
+ * Initializing
+ */
+
+const fetchTicketGroups = () =>
+  fetch('/examples/data/ticket-groups-1591449.json')
+    .then(response => response.json())
+
+const buildMap = () => {
+  map = new Tevomaps({
+    venueId: '896',
+    configurationId: '14341',
```

Comment:
Would you mind defining these values as constants at the top of the file? I tend to fear large files with magic values sprinkled about.